starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,53 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Arial;
+}
+
+header {
+  text-align: center;
+  height: 600px;
+  width: 100vw;
+  background-image: url("../images/apple-pie.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin-bottom: 50px;
+}
+
+h1 {
+  position: absolute;
+  top: 37%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: white;
+  font-size: 4em;
+  font-family: Arial;
+  font-weight: 300;
+}
+
```

Comment:
I would suggest  avoid using h1 as much as possible as it is better practice to use a class instead.

Reply:
didn't know that, thanks!